pkgs/applications/networking/seafile-client/default.nix

```diff
@@ -1,5 +1,8 @@
 { stdenv, fetchurl, writeScript, pkgconfig, cmake, qtbase, qttools
-, seafile-shared, ccnet, makeWrapper }:
+, seafile-shared, ccnet, makeWrapper
+, withShibboleth ? false, qtwebengine }:
```

Comment:
You can default to `true`, if this feature does not introduce a regression.

Reply:
Well I think I'll use `true`then but I'll keep the option since it's also available upstream.